boards/st/nucleo_wba55cg/nucleo_wba55cg.dts

```diff
@@ -62,6 +62,18 @@
 		};
 	};
 
+	die_temp: dietemp {
+
+		compatible = "st,stm32-temp-cal";
+		ts-cal1-addr = <0x0BF90710>;
+		ts-cal2-addr = <0x0BF90742>;
+		ts-cal1-temp = <30>;
+		ts-cal2-temp = <130>;
+		ts-cal-vrefanalog = <3000>;
+		io-channels = <&adc4 13>;
+		status = "okay";
+	};
+
```

Comment:
This should be in the .dtsi file (except that the status should be `"disabled"` in it).
You should only have `status = "okay";` here.